frontend/components/SinglelineTextControl/_index.html.erb

```diff
@@ -14,6 +14,6 @@
     name="<%= id %>"
     id="<%= id %>"
     type="text"
-    size="<%= defined?(size) ? size : "" %>"
+    size="<%= defined?(size) ? size : '' %>"
```

Comment:
This could be rewritten as `<?= size if defined?(size) %>` which is in keeping with how we've done it on line 9